packages/wallet-sdk/src/CoinbaseWalletProvider.test.ts

```diff
@@ -32,7 +32,6 @@ describe('CoinbaseWalletProvider', () => {
     test('throws error when handling invalid request', async () => {
       const provider = createProvider();
       // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-expect-error // testing invalid request args
```

Comment:
we don't know what dapps would pass, and that's the whole point of us having `checkErrorForInvalidRequestArgs` assertion.
we actually should not expect error here.